test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/main.py

```diff
@@ -306,7 +306,7 @@ class ModuleValidator(Validator):
     # win_dsc is a dynamic arg spec, the docs won't ever match
     PS_ARG_VALIDATE_REJECTLIST = frozenset(('win_dsc.ps1', ))
 
-    ACCEPTLIST_FUTURE_IMPORTS = frozenset(('absolute_import', 'division', 'print_function'))
+    ACCEPTLIST_FUTURE_IMPORTS = frozenset(('absolute_import', 'annotations', 'division', 'print_function'))
```

Comment:
Should this be Python-version dependent? The modules that execute on the target node, could be executed with an old Python interpreter that would cause an `ImportError`. IIRC Python 3.6 and below would do this for `annotations`.

Reply:
Potentially but that would most likely fail in other tests, the check here doesn't enforce that `annotations` is present, just allows it to be in the imported values. Doing a version dependent check would mean it would complicate the test to add in the extra checks to make sure `annotations` is only done on the controller side.

Even then you could potentially have collections which have modules or module_utils that are Python 3.7+ only and they want to enable text annotations.